projectforum/static/scss/base.scss

```diff
@@ -8,6 +8,7 @@
 @import 'project_detail';
 @import 'project_list';
 @import 'user_profile';
+@import 'bookmarks';
```

Comment:
This will need to be moved to a new folder if you pull from master. Also maybe consider renaming to project_bookmarks?